css/css-scroll-snap/scroll-target-margin-003.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+<meta charset="utf-8">
+<title>focus() and scroll-margin with snapping off (y-axis)</title>
+<link rel='author' title='Elika J. Etemad' href='http://fantasai.inkedblade.net/contact'>
+<link rel='help' href='https://www.w3.org/TR/css-scroll-snap-1/#scroll-margin'>
```

Comment:
Same comment as scroll-target-align-003.html: I agree the behavior is desirable with `focus()` too, but is that supported by the spec?

Reply:
Same reply.